src/toolbars/SelectionBar.cpp

```diff
@@ -167,42 +175,103 @@ void SelectionBar::Create(wxWindow * parent)
    UpdatePrefs();
 }
 
+AButton* SelectionBar::MakeSetupButton()
+{
+   wxImage up = theTheme.Image(bmpRecoloredUpSmall);
+   up.Rescale(20, 20, wxIMAGE_QUALITY_HIGH);
+   wxImage down = theTheme.Image(bmpRecoloredDownSmall);
+   down.Rescale(20, 20, wxIMAGE_QUALITY_HIGH);
+   wxImage hiliteUp = theTheme.Image(bmpRecoloredUpHiliteSmall);
+   hiliteUp.Rescale(20, 20, wxIMAGE_QUALITY_HIGH);
+   wxImage hiliteDown = theTheme.Image(bmpRecoloredHiliteSmall);
+   hiliteDown.Rescale(20, 20, wxIMAGE_QUALITY_HIGH);
+   
+   auto btn = safenew AButton(
+      this, wxID_ANY, wxDefaultPosition, wxSize { 20, 20 }, up, hiliteUp, down,
+      hiliteDown, up, false);
+   
+   btn->SetButtonType(AButton::FrameButton);
+   btn->SetIcon(theTheme.Image(bmpCogwheel));
+   btn->SetLabel({});
+   btn->SetName(XO("Selection Toolbar Setup").Translation());
+
+   return btn;
+}
 
-auStaticText * SelectionBar::AddTitle(
+void SelectionBar::AddTitle(
    const TranslatableString & Title, wxSizer * pSizer ){
    const auto translated = Title.Translation();
    auStaticText * pTitle = safenew auStaticText(this, translated );
    pTitle->SetBackgroundColour( theTheme.Colour( clrMedium ));
    pTitle->SetForegroundColour( theTheme.Colour( clrTrackPanelText ) );
    pSizer->Add( pTitle, 0, wxEXPAND | wxRIGHT, 5 );
-
-   return pTitle;
 }
 
 
-NumericTextCtrl * SelectionBar::AddTime(
-   const TranslatableString &Name, int id, wxSizer * pSizer ){
+void SelectionBar::AddTime(
+   int id, wxSizer * pSizer ){
    auto formatName = mListener ? mListener->AS_GetSelectionFormat()
       : NumericFormatSymbol{};
    auto pCtrl = safenew NumericTextCtrl(
       this, id, NumericConverter::TIME, formatName, 0.0, mRate);
-   pCtrl->SetName( Name );
```

Comment:
The `SetName` call got lost and that has bad implications for the screen reader.

Reply:
That's unfortunate.

Reply:
I have rechecked and this change is correct. The name is now set in `SelectionBar::SetSelectionMode`. This method is called from `SelectionBar::Populate()` so the name is always set and correct.

Reply:
I'm satisfied